config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
    get "/", to: "welcome#index"
 
+
+   get '/profile', to: "users#show"
```

Comment:
Is this route ok, or did u want a separate profile/dashboard controller instead

Reply:
Looks good to me.

Reply:
looks good to me also